### ISSUE TITLE
Explore the possibility of using a new set of fonts for the site

### ISSUE THREAD

**Issue by oubiwann** [lfe/docs#88]
The following site was pointed out today on the LFE  #web Slack channel:
 * https://www.awwwards.com/20-best-web-fonts-from-google-web-fonts-and-font-face.html

In particular, the following caught my eye:
 * https://fonts.google.com/?query=josefin

Let's try these out and see if they're a good match for the LFE site, and if so, we can add this font to the style guide (#53)

Part of epic #66

**Comment by oubiwann** [comment]
Notes on the Josefin Sans:

<blockquote>
The idea for creating this typeface was to make it geometric, elegant and kind of vintage, especially for titling. It is inspired by Rudolf Koch's Kabel (1927), Rudolf Wolf's Memphis (1930), Paul Renner's Futura (1927).

My idea was to draw something with good style, that reflects Swedish design and their passion for a good lifestyle, and by default all other Scandinavian styles.

Something weird happened when I drew the letter "z": when I took my typography courses in college, I saw a very interesting typeface in a book, the New Universal Typeface "Newut" from André Baldinger (which I have loved since then), and after I finished the "z" I ran into Newut' site again and noticed that I had unconsciously drawn it with the same haircut.

The x-height is half way from baseline to caps height, unlike any other modern typeface.

I wanted to do something different with the ampersand, so I made three and will include them as alternates in future versions. This version also includes Old Style Numerals.

There is a sister family, Josefin Slab.

Updated August 2014: Completely redesigned and extended with Extended Latin support for many more European languages.

The previous version of Josefin Sans included Extended Latin characters in the Latin subset. If you are experiencing problems, please select the Extended Latin (latin-ext) subset to render all European languages correctly. Here is an example of a complete link tag that selects these subsets:
</blockquote>

```html
<link href='http://fonts.googleapis.com/css?family=Josefin+Sans&subset=latin,latin-ext' 
      rel='stylesheet' type='text/css'>
```

(From https://fonts.google.com/specimen/Josefin+Sans?category=Serif&selection.family=Josefin+Sans)

**Comment by oubiwann** [comment]
Notes on the Josefin Slab:

<blockquote>
Josefin Slab was the first typeface–at least in my mind–I designed! But I decided to start simple with Josefin Sans. Following the 1930s trend for geometric typefaces, it just came to me that something between Kabel and Memphis with modern details will look great.

I wanted to stick to the idea of Scandinavian style, so I put a lot of attention to the diacritics, especially to "æ" which has loops connecting in a continuous way, so the "e" slope was determined by this character.

It also has some typewriter style attributes, because I've liked the Letter Gothic typeface since I was in high school, and that's why I decided to make a Slab version of Josefin Sans.
</blockquote>

(From https://fonts.google.com/specimen/Josefin+Slab?category=Serif&selection.family=Josefin+Slab)
